jena-fuseki2/jena-fuseki-war/pom.xml

```diff
@@ -62,14 +70,19 @@
           </execution>
         </executions>
         <configuration>
-          <warSourceDirectory>../jena-fuseki-webapp/src/main/webapp</warSourceDirectory>
+          <warSourceDirectory>../jena-fuseki-ui/target/dist/</warSourceDirectory>
```

Comment:
Shouldn't this be `jena-fuseki-webapp/target/webapp/`

If've been doing some questionable things with Maven and WARs in $dayjob lately and even I'm having trouble following this

Reply:
@rvesse 

>Shouldn't this be jena-fuseki-webapp/target/webapp/

Ah! You are correct. Yesterday I jotted down a quick diagram of the new build hierarchy, and the `webapp` module now includes the contents of the `ui` module plus some other files.

![20211125_112226](https://user-images.githubusercontent.com/304786/143320379-59f9c409-2c92-4cc4-90ed-2c521865f0eb.jpg)

>If've been doing some questionable things with Maven and WARs in $dayjob lately and even I'm having trouble following this

:rofl: